runtime/doc/news.txt

```diff
@@ -259,6 +259,9 @@ The following changes to existing APIs or features add new behavior.
   Note that syntax highlighting of code examples requires a matching parser
   and may be affected by custom queries.
 
+• |nvim_open_win()| and |nvim_win_set_config()| now support opening normal (split)
+  windows, and moving floating windows into split windows.
```

Comment:
Does the reverse also work (normal => floating)?

Reply:
Yes